agent/dockerclient/dockerapi/docker_client_integ_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build integration
-// +build integration
+//go:build !windows && integration
```

Comment:
I think we should rename this file to have `unix` in its name to follow the convention.